wemake_python_styleguide/visitors/ast/subscripts.py

```diff
@@ -182,3 +183,72 @@ def _is_float_key(self, node: ast.expr) -> bool:
         return isinstance(real_node, ast.Constant) and isinstance(
             real_node.value, float
         )
+
+
+@final
+class StricterSliceOperations(base.BaseNodeVisitor):
+    """Check for stricter operation with slices."""
+
+    def visit_Slice(self, node: ast.Slice):
+        """Check that there is no stricter way to use slices."""
+        self._check_reverse_through_slice(node)
+        self._check_copy_through_slice(node)
+        self._check_pop_through_slice(node)
+        self.generic_visit(node)
+
+    def _check_reverse_through_slice(self, node: ast.Slice):
```

Comment:
```suggestion
    def _check_reverse_through_slice(self, node: ast.Slice) -> None:
```

(and all other places)